src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -63,6 +63,22 @@ The API used to retrieve the value is:
   The number of times garbage collection has occurred for the specified generation
 of objects.
 
+#### process.runtime.dotnet.**gc.objects.size**
+
+Count of bytes currently in use by objects in the GC heap that haven't been
+collected yet.
+Fragmentation and other GC committed memory pools are excluded.
+
+| Units   | Instrument Type         | Value Type | Attribute Key(s)  | Attribute Values |
+|---------|-------------------------|------------|-------------------|------------------|
+| `bytes` | ObservableUpDownCounter | `Int64`    | No Attributes     | N/A              |
+
+The API used to retrieve the value is:
+
+* [GC.GetTotalMemory](https://docs.microsoft.com/dotnet/api/system.gc.gettotalmemory):
+  Retrieves the number of bytes currently thought to be allocated.
+The value is an approximate count.
```

Comment:
would be nice to indicate that this value is not the snapshot from the last GC event, and also that this metric will be available irrespective of if a GC event has occured or not.

Reply:
added in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/772/commits/bd9b48768a89ef6c4d0c055fb85dd2a2898898a3